styles/styles.css

```
@@ -266,6 +266,22 @@ em strong {
   font-style: italic;
 }
 
+.video-wrapper {
+    margin: 40px auto;
+    max-width: 700px;
+}
+
+p.video-title {
+  font-size: 24px;
+  color: #34393d;
```

Should this be replaced by the variable?

```suggestion
  color: var(--text-color);
```